libc/src/stdio/generic/fileno.cpp

```diff
@@ -0,0 +1,22 @@
+//===-- Implementation of fileno
+//-------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/stdio/fileno.h"
+#include "src/__support/File/file.h"
+
+#include <stdio.h>
```

Comment:
Please don't include <stdio.h> for hermeticity; that may be the system's headers rather than llvmlibc's.

You should probably be including "include/llvm-libc-types/FILE.h" instead.

Reply:
Updated.